nntrainer/tensor/blas_interface.cpp

```diff
@@ -400,12 +381,64 @@ void scopy_int8_to_float16(const unsigned int N, const uint8_t *X,
   copy_int8_to_fp16(N, X, incX, Y, incY);
 }
 
-void ewvm(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z) {
-  ewvm_FP16(N, X, Y, Z);
+void ele_mul(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z,
```

Comment:
```suggestion
void elementwise_multiply(const unsigned int N, const _FP16 *X, const _FP16 *Y, _FP16 *Z,
```
one suggestion. how about renaming it into something more clear?

Reply:
I asked for many contributors' opinions offline, but shorter function name is quite preferred.
ele_* sounds clear enough for me.